### DISCUSSION TITLE
Sprite depth of thalamus items

### DISCUSSION THREAD

**Discussion by frijoles108** [FakeFishGames/Barotrauma#14863] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Ive noted that when i take the cargo scooter into wrecks, it will disappear when i drop it from hand to grab something else in a cabinet

### Reproduction steps

Go into wrecked sub, possible near destroyed door? And drop scooter. I saw a similar issue happening to someone else in alien ruins. 

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by jcx1991911** [comment]
![20250116235339_1](https://github.com/user-attachments/assets/94ee7512-05d3-435b-9040-9d40c12cf751)
![20250116235410](https://github.com/user-attachments/assets/84e8a3b1-96c4-4a49-9cf7-7bdd216c1569)
Because these organs have a sprite depth of 550 or less, most equipment will be covered by it if it's lucky enough to be dropped here

**Reply by jcx1991911** [reply]
But if it's “physically” disappear, then we may not be talking about the same thing I guess...

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in the next unstable build and the next update.
